app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class VotesController < ApplicationController
+
+  def create
+    current_user.votes.create(votable_type: vote_params[0], votable_id: vote_params[1])
```

Comment:
Ah I didn't know about about the `require` method. so here, we could have something like

```
def create
  votable_type, votable_id = vote_params
  current_user.votes.create(votable_type: votable_type, votable_id: votable_id)
end
```

Though, I think my vote (pun intended) would eventually be nesting `type` and `id` under `votable` and writing a `params.require(:votable).permit(:id, :type)`.